src/renderer/views/Watch/Watch.js

```diff
@@ -376,7 +376,7 @@ export default defineComponent({
 
         // extract localised title first and fall back to the not localised one
         this.videoTitle = result.primary_info?.title.text ?? result.basic_info.title
-        this.videoViewCount = result.basic_info.view_count ?? extractNumberFromString(result.primary_info.view_count.text)
+        this.videoViewCount = result.basic_info.view_count ?? (result.primary_info.view_count ? extractNumberFromString(result.primary_info.view_count.text) : 0)
```

Comment:
What's the reason for defaulting to 0 instead of null if the ternary starement it's false?

Reply:
watch-video-info expects there to always be a view count, so this was the easiest way to get it to work. That being said you are right that the correct approach would be make it optional like we do in other parts of the app, I'll do that in this PR.

Reply:
Done